### ISSUE TITLE
UH DK Opener Suggestions Not Following SimC

### ISSUE THREAD

**Issue by Ullti** [Hekili/hekili#726]
Snapshot of UH DK ST Opener:  https://pastebin.com/awnLGxHw

Quick Sim of my UH DK Opener:  https://www.raidbots.com/simbot/report/vkwdWRKuoSQc58YSwDTX6S

Image of 10 Q on Hekili Targeting a Dummy:  ![image](https://user-images.githubusercontent.com/76457144/102832787-45d1b180-43a4-11eb-8767-842c7eefeaa5.png)

Suggested Opener from UH DK Discord FAQ:  ![image](https://user-images.githubusercontent.com/76457144/102832851-71ed3280-43a4-11eb-9794-211023fd257a.png)

So currently the addon suggests that I do the following:
Army>Blight>DT>Festering>DC>Racial>DC>Festering>Apoc>Cov

However, the Discord community and my sim suggest I do the following:
Army>Festering>Blight>Cov>DT>Festering>Apoc+Racial(But I'm a BE)

As mentioned in the screenshot from the FAQ the placement of Festering is marginal at best.

**Comment by Hekili** [comment]
So, the sim:
![image](https://user-images.githubusercontent.com/9438134/102834211-64927100-43c1-11eb-9307-6da88dca6ed9.png)

That is:
Army > Blight > Abom Limb > Trinket > Dark Trans > Death Coil > Festering > Festering > Apocalypse > Scourge

The addon:
Army > Blight > Dark Trans > Festering > Death Coil > Arcane Torrent > Festering > Apocalypse > Abom Limb

You're right that it doesn't seem to be detecting your Soulbinds.  If you go to /hekili > Issue Reporting, do you see them listed next to `conduits:` and `soulbinds:` ?  I can see that you have Emeni selected but the addon didn't detect any of the soulbinds or conduits there.

That seems as though it'd move up Abom Limb effectively, but I'll need more details before I can know why your Conduits/Soulbinds didn't load.

Do they show in /hek > Issue Reporting after your first login?
After /reload?
After swapping Soulbinds back and forth?

**Comment by Ullti** [comment]
So after reloading I am seeing it list Emeni as a Soulbind with the conduits and everything.  I tried simply closing the client and rebooting previously and did not get this same result.  I am also seeing the APL make the correct suggestions when looking at the Q.

Here is a link to what I see when looking at the character data.
https://pastebin.com/ZfdcGYYw

I will have to find a way to replicate when my Soulbind begins to go undetected.

One that I can remember off the top of my head was when loading into a mythic plus earlier in the day.  It also just disappeared when I hearthed to my Covenant Hall. Conduits are gone but it recognizes my soulbind.

![image](https://user-images.githubusercontent.com/76457144/102835275-1e321780-43ab-11eb-939a-70b6cbf0d3a0.png)
